### PR TITLE
[IA-3953] Added status text to the error response for convert and convert_rmd.

### PR DESCRIPTION
**Pull request by n8landolt**
Adding status text will allow us to capture the error information in MixPanel, which should hopefully give us better insight into the issues users are having with preview.

For those unfamiliar with flask, the 400 in the return will be parsed as the status, and the rest of the string will be returned in the statusText value of the response.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -59,7 +59,9 @@ def convert():
     try:
       return perform_notebook_conversion(json)
     except Exception as e:
-      return render_template('jupyter-error.html', error=e.__class__.__name__ + ": " + str(e) + ". ") ,400
+      errMessage = e.__class__.__name__ + ": " + ''.join(str(e).splitlines()) + "."
```
Comment (LizBaldo) [inline main.py]: Nit, it could be nicer to use an f string here instead of `+` but really not a blocker

**Review by LizBaldo — APPROVED** [review]

File: main.py (hunk 1 above)
Comment (lucymcnatt) [reply main.py]: was gonna suggest that as well!

**Review by lucymcnatt — APPROVED** [review]

File: main.py [hunk 2]
```diff
@@ -59,7 +59,9 @@ def convert():
     try:
       return perform_notebook_conversion(json)
     except Exception as e:
-      return render_template('jupyter-error.html', error=e.__class__.__name__ + ": " + str(e) + ". ") ,400
+      errMessage = f'{e.__class__.__name__} : {''.join(str(e).splitlines())} .'
```
Comment (lucymcnatt) [inline main.py]: I think using the single quotes here in the .join are messing with the f string

File: main.py [hunk 3]
```diff
@@ -69,7 +71,8 @@ def convert_rmd():
     try:
       return perform_rmd_conversion(stream)
     except Exception as e:
-      return render_template('rstudio-error.html', error=e.__class__.__name__ + ": " + str(e) + ". ") ,400
+      errMessage = f'{e.__class__.__name__} : {''.join(str(e).splitlines())} .'
```
Comment (lucymcnatt) [inline main.py]: same here

File: main.py (hunk 2 above)
Comment (n8landolt) [reply main.py]: I don't know what you're talking about 👀 😜

**Pull request merged**